components/tab-tile/tab-tile.scss

```diff
@@ -12,26 +18,74 @@
 
 .msds-tab-tile {
   display: flex;
+  flex: 1;
   flex-direction: column;
   align-items: center;
-  min-width: 277px;
-  height: 180px;
-  border: 1px solid $gray-4;
-  background: transparent;
+  margin-right: 4px;
+  background: $white;
+  max-width: 371px;
+  border-top: 4px solid $white;
+  border-left: 1px solid $white;
+  border-right: 1px solid $white;
   @include msds-font('body-1');
   font-weight: bold;
+  text-align: center;
   color: $gray-9;
   cursor: pointer;
-  padding-top: 24px;
+  padding: 20px 16px 24px 16px;
+
   svg {
     width: 120px;
     height: 90px;
     margin-bottom: 20px;
   }
 
+  &__text {
+    max-width: 200px;
+  }
+
+  &:hover {
+    color: $clear-blue;
+  }
+
+  &:focus {
+    outline: 4px solid rgba(0, 153, 218, 0.5);
+    color: $clear-blue;
+  }
+
   &--active {
+    border: 1px solid $gray-4;
     background: $white;
     border-top: 4px solid $clear-blue;
     border-bottom: none;
+    color: $clear-blue;
+  }
+
+  &--disabled {
+    color: $gray-7;
+    filter: grayscale(1);
```

Comment:
doesn't look to have widespread support?
https://caniuse.com/#search=grayscale

Reply:
No IE 11 is not really happy with the grayscale, ive added correct colours to all other elements on the tab, so CE will have to supply us with the  correct image when building the block